GameData/KerbalHacks/DropTanks/Localization/nl-nl.cfg

```diff
@@ -1,35 +1,46 @@
-		
-		#DROP-Bracelet-125-titl = 1.25m Droptank "Armband"
-		#DROP-Bracelet-125-desc = Voor degenen die van hun kleine en schattige raketten houden. Crossfeed is schakelbaar.
-
-		#DROP-Wristband-125-titl = 1.25m Droptank "Polsbandje"
-		#DROP-Wristband-125-desc = Toen het ontbreken van kortere Wikkel-tanks een rel veroorzaakte onder de raketbouwers, besloot Wherner het huidige model doormidden te snijden om de raketbouwers tevreden te stellen. Crossfeed is schakelbaar.
-	}
+// nl-nl.cfg v1.0.1.2
+// Drop Tanks (DROP)
+// created: 19 Dec 2021
+// updated: 22 Jan 2023
+
+// THIS FILE:
+//   CC BY-NC-SA 4.0 by zer0Kerbal and renejant
+//   generated by [zer0Kerbal](https://github.com/zer0Kerbal)
+//   translated courtesy of  [renejant](https://github.com/renejant) *bedankt!*
+
+Localization
+{
+	nl-nl
+	{
+	// KerbalHacks\DropTanks
+		#DROP-mod-name = Drop Tank Wikkel
+		#DROP-tags = drop wrap enceos wikkel dop mouw armband polsbandje
+	
+	// Parts
+		#DROP-Cap-125-titl = 1.25m Droptank "Aerodynamische Dop"
+		#DROP-Cap-125-desc = Na vele verzoeken om het lelijke uiteinde van de Wikkel-tanks te verbergen voor de luchtstroom, bedacht onze engineeringafdeling deze prachtige dop. We zijn er zelfs in geslaagd om een kleine brandstoffles erin te plaatsen.
+
+		#DROP-Wrapper-125-titl = 1.25m Droptank "Wikkel"
+		#DROP-Wrapper-125-desc = We verzamelden wat er over was na een ongelukkige Jumbo-prototype-explosie en dachten "waarom zou ik het verspillen?". Dus namen we een ducttape en bonden er wat ontkoppelbouten op vast. Valentina noemde het "Wikkel", omdat het eruitzag alsof de raket ingewikkeld was met folie. Crossfeed is schakelbaar.
+
+		#DROP-Sleeve-125-titl = 1.25m Droptank "Mouw"
+		#DROP-Sleeve-125-desc = De recente explosie van het "Jumbo"-prototype was niet zo groot als de eerste, waardoor we met langere segmenten moesten werken. Duct tape magie, klinknagels en ontkoppelingsbouten presenteerden ons een nieuwe droptank. We noemden het "Mouw", het was vrijwel zeker langer dan het eerste prototype en leek sterk op de mouw van Wherner's laboratoriumjas als het eromheen werd gewikkeld. Crossfeed is schakelbaar.
+		
+		#DROP-Bracelet-125-titl = 1.25m Droptank "Armband"
+		#DROP-Bracelet-125-desc = Voor degenen die van hun kleine en schattige raketten houden. Crossfeed is schakelbaar.
+
+		#DROP-Wristband-125-titl = 1.25m Droptank "Polsbandje"
+		#DROP-Wristband-125-desc = Toen het ontbreken van kortere Wikkel-tanks een rel veroorzaakte onder de raketbouwers, besloot Wherner het huidige model doormidden te snijden om de raketbouwers tevreden te stellen. Crossfeed is schakelbaar.
+
+	// B9partswitch - WARNING: This section done with Google Translate - WAARSCHUWING: dit gedeelte gedaan met Google Translate

```

Comment:
@help137 
nice! appreciate this detail. 👍 🦃 🧧 

same as the en-us version. :D